src/pages/update-browser.tsx

```diff
@@ -6,14 +6,17 @@ import Logo from 'components/Logo/Logo'
 import LinkTo from 'components/util/LinkTo/LinkTo'
 
 import styles from 'styles/pages/updateBrowser.module.scss'
-
+import Head from 'next/head'
 export default function UpdateBrowser() {
   const MS_EDGE_DOWNLOAD = 'https://www.microsoft.com/edge'
   const GOOGLE_CHROME_DOWNLOAD = 'https://www.google.com/chrome'
   const FIREFOX_DOWNLOAD = 'https://www.mozilla.org/firefox'
 
   return (
     <>
+      <Head>
+        <title>Browser Warning - USSF Portal</title>
+      </Head>
```

Comment:
Suggestion: remove the `head` tag, looks like this page has both a head tag and a `pageTitle` prop.